### PR TITLE
TF-1472: Fixed popup when sending email should be displayed at the bottom of screen

### PR DESCRIPTION
**Pull request by nqhhdev**
Issue
#1472

Resolved 

https://user-images.githubusercontent.com/99852347/221746283-c931af97-be51-4509-9dba-85180aa7b827.mp4

### CODE REVIEW

**Review by hoangdat — APPROVED** [review]

File: lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart [hunk 1]
```diff
@@ -253,10 +253,11 @@ class MailboxDashBoardController extends ReloadableController {
       (failure) {
         log('MailboxDashBoardController::onData():failure $failure');
       },
-      (success) {
+      (success) async {
         log('MailboxDashBoardController::onData():success $success');
         if (success is SendingEmailState) {
           if (currentOverlayContext != null && currentContext != null) {
+            await Future.delayed(const Duration(seconds: 1));
```
Comment (dab246) [inline lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart:260]: Should add option `delayTime` in `showToast` function to reuse for more case. Ex:
```
_appToast.showToastWithIcon(
    currentOverlayContext!,
    message: AppLocalizations.of(currentContext!).your_email_being_sent,
   delayTime: const Duration(seconds: 1)
);
```

Comment (nqhhdev) [reply lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart:260]: Greattt, thanks u

File: core/lib/presentation/utils/app_toast.dart [hunk 2]
```diff
@@ -181,6 +182,7 @@ class AppToast {
       ),
     );
     fToast.init(context);
+    await Future.delayed(delayTime ?? const Duration(seconds: 0));
```
Comment (dab246) [inline core/lib/presentation/utils/app_toast.dart:185]:
```suggestion
if (delayTime != null) {
    await Future.delayed(delayTim));
}
```

File: lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart [hunk 3]
```diff
@@ -253,14 +253,16 @@ class MailboxDashBoardController extends ReloadableController {
       (failure) {
         log('MailboxDashBoardController::onData():failure $failure');
       },
-      (success) {
+      (success) async {
```
Comment (dab246) [inline lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart:256]: remove `async`

**Review by dab246 — APPROVED** [review]

**Pull request merged**